scripts/generatePagesFromConsolidatedJSON.py

```diff
@@ -13,6 +17,15 @@ def genChartHeader(f):
     """)
 
 
+def isGRSI(title, GRSI):
+    for titleGRSI in GRSI:
+      seq = SequenceMatcher(None, title.upper(), titleGRSI.upper())
+      if seq.ratio() > 0.85:
```

Comment:
I'm curious: was this threshold easy to set ?
Shall we print another message if we get slightly lower values, just in case we missed something ?

Reply:
the two strings are super close. I think I could have done an exact match. 
0.85 is the THE magic threshold ;)

Reply:
This is quite temporary.  In the future, we should sync from the DOI with the GRSI database. At this point, they do not "expose" some csv/json public data